src/main/java/goodgenerator/loader/Loaders.java

```diff
@@ -111,7 +111,11 @@ public class Loaders {
                 GoodGenerator.MOD_ID + ":ciruits/4",
                 GoodGenerator.MOD_ID + ":ciruits/5"
             });
-
+    public static final Item nuclearStar = new MyItems(
```

Comment:
Wouldn't it be better to put that into GT5U or coremod? That way we could easily use it in any mod, since I'd think we want to replace the gravistar in normal assembling line crafts for UIV+ too.